src/observers/docsLoggerObserver.ts

```diff
@@ -153,9 +153,13 @@ export class DocsLoggerObserver {
         } else {
             repositoryUrl = event.localRepositoryUrl;
         }
+        let branch = event.repositoryBranch;
+        if (branch === 'HEAD') {
+            branch += `(Commit: ${event.commit.substr(0, 6)})`;
+        }
```

Comment:
Can always print the commit for easier trouble shooting

Reply:
Actually I am thinking to directly remove the branch and commit info for that:
1. branch info is not necessary to run validation(Now we pass the branch info as an environment variable to docfx)
2. commit info may bring confusion to the user that the validation is running against the commit, but actually it is not, it is running against the current content, including the uncommitted content.

 @yufeih @herohua @live1206 Any idea?

Reply:
`docfx` itself already computes the correct repository url and branch internal, thus I agree there is no need to compute branch and commit here at all.

`repositoryUrl` is needed because translating fork is currently done in vscode extension. Though personally I felt strange that building from vscode and running `docfx build` from the command line produces different result.

Reply:
@mingwli , for your concern, I think we can print a log in docfx

Reply:
@yufeih 
Since `docfx` already computes the git info, why were `DOCS_REPOSITORY_BRANCH` and `DOCS_REPOSITORY_URL` needed in the first place?

Reply:
As I mentioned, printing commit may bring confusion

---
In reply to: [380126915](https://github.com/docascode/vscode-docs-build/pull/46#discussion_r380126915) [](ancestors = 380126915)

Reply:
For your concern, My opinion is we should try to keep consistent with server build result(or PR result) instead of docfx.

---
In reply to: [380126670](https://github.com/docascode/vscode-docs-build/pull/46#discussion_r380126670) [](ancestors = 380126670)

Reply:
> @mingwli , for your concern, I think we can print a log in docfx

As I mentioned, printing commit may bring confusion, @mingwli Can you give more detail about your idea

Reply:
> > @mingwli , for your concern, I think we can print a log in docfx
> 
> As I mentioned, printing commit may bring confusion, @mingwli Can you give more detail about your idea

I was just thinking that since current commit hash is computed, why not just simply print it out.  
(e.g. if user is working on a certain working branch but forget to pull latest content, which leads to different warnings with server build, this can be easily identified by knowing which commit user is building against).  

While I didn't think about that this could confuse the users.   
I agree that should keep the logger output as simple as possible.